### PR TITLE
Improve inspect page

### PR DESCRIPTION
**Pull request by shu8**
This PR:

- wires up the import buttons to update the UI on import
- wires up the update manifest button to actually update it
- moves the import flow to a separate screen
- finished wiring up the bake button
- UX improvements
  - 'trickling' progress bars during long tasks
  - rename Export tab to Actions
  - disable the 'add file' button until a file is uploaded

A demo of what it looks like now:

inspect-demo

### CODE REVIEW

**Comment by avaer** [comment]
Just gotta say, screencaps in PRs are amazing.

**Comment by avaer** [comment]
Is this branch up to date with `master`? I still see the weird design.

Screen Shot 2020-07-12 at 6 09 24 AM

**Comment by avaer** [comment]
Screen Shot 2020-07-12 at 6 10 18 AM

This states that you can browse packages to inspect but you actually can't do that from the browse page. Imo the browse page needs major improvements, one of them being the ability to inspect packages intuitively (right now the UI for moving forward from a package to the next page is really confusing).

File: inspect.js [hunk 1]
```diff
@@ -0,0 +1,526 @@
...
+        }
+      };
+      window.addEventListener('message', _message);
+    });
+    iframe.addEventListener('error', err => {
+      reject(err);
+    });
+  });
+
+  // Baking logic from xrpk CLI: https://github.com/webaverse/xrpackage-cli/blob/master/commands/bake.js
```
Comment (avaer) [inline inspect.js:299]: I don't know if I would reference files like this in comments. Chances are good that file won't exist in a month.

File: progress.js [hunk 2]
```diff
@@ -34,9 +35,40 @@ class Progress {
       nanobar.go(100);
     }
   }
+  trickle() {
```
Comment (avaer) [inline progress.js:38]: Good addition!

**Comment by shu8** [comment]
Sounds good thanks -- was just about to rebase 😅

**Pull request merged**